integration/tests/conftest.py

```diff
@@ -39,7 +40,11 @@ def pytest_collection_modifyitems(config, items):
     settings = network_manager.get_network_object(network_name)
     web3_client = web3client.NeonChainWeb3Client(settings["proxy_url"])
 
-    raw_proxy_version = web3_client.get_proxy_version()["result"]
+    if network_name == "geth":
+        raw_proxy_version = "10000000000000"  # don't deselect any tests for geth based on the proxy version
```

Comment:
I don't understand this comment

Reply:
in this hook, tests that have @pytest.mark.proxy_version(<version>) lower than the proxy version in the environment, are deselected
so this was making sure that no tests are deselected based on this logic

I've read the rest of the code in this hook and it looks like it's all irrelevant for geth env, and we can just `return` if the network is geth